COMPILING.md

```diff
@@ -87,9 +87,17 @@ git clone --recursive https://github.com/cuberite/cuberite.git
 
 ### Building ###
 
-Follow the instructions at [CMake on Unix-based platforms](#cmake-on-unix-based-platforms).
+Run the following commands to build Cuberite:
+
+```
+mkdir Release
+cd Release
+cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+make -j`nproc`
```

Comment:
Make -j3 causes pretty much any machine to run out of RAM. -j2 or -j3 is probably optimal for most machines (IO wait is minimised on single threaded machines, and becomes the limiting factor with more threads).